src/main.js

```diff
@@ -19,6 +19,8 @@ import Payment from './pages/Payment.vue';
 import Login from './pages/Login.vue';
 import User from './pages/User.vue';
 
+console.log(BUILDDATE + ' ' + VERSION + ' ' + BRANCH);
+
```

Comment:
Also it could possibly go in the main app mounted method: https://github.com/neontribe/ARCVMarket/pull/126/files/51d029d102133af4a1c276a49e5671a1a503a8e4..60d07664adab306cb2a8588a82be47d266af50ae#diff-1c90ff38a08209f9ebd4d05d1e43358eL98